src/pages/index.js

```diff
@@ -54,39 +72,136 @@ const profileEditModal = new ModalWithForm(
   handleProfileEditSubmit
 );
 profileEditModal.setEventListeners();
+
 const addCardModal = new ModalWithForm("#add-card-modal", handleAddCardCreate);
 addCardModal.setEventListeners();
 
+const profileAvatarEditModal = new ModalWithForm(
+  "#profile-avatar-edit-modal",
+  handleProfileAvatarUpdate
+);
+profileAvatarEditModal.setEventListeners();
+
+// Add an event listener to the button to open the avatar update modal
+const profileAvatarEditButton = document.querySelector(
+  "#profile-avatar-edit-button"
+);
+profileAvatarEditButton.addEventListener("click", () => {
+  profileAvatarEditModal.open();
+  profileAvatarInput.value = "";
```

Comment:
* You should **clear ** the inputs only after **submitting** in `then` a card rather than when you open or close popups because if you unexpectedly close the popup, you will be upset if the inputs are empty when you open it again, and you’ll have to type the data again.


**Please, correct it everywhere in the code**